tasks.py

```diff
@@ -1556,10 +1556,10 @@ def version(c):
 Backup      {get_backup_dir(error=False) or NOT_SPECIFIED}
 
 Versions:
-Python      {python_version()}
-Django      {InvenTreeVersion.inventreeDjangoVersion()}
 InvenTree   {InvenTreeVersion.inventreeVersion()}
 API         {InvenTreeVersion.inventreeApiVersion()}
+Python      {python_version()}
+Django      {InvenTreeVersion.inventreeDjangoVersion()}
```

Comment:
Was this change intended to be here?

Reply:
Yes, it's not strictly related but I think it's better to have our version first